components/cascader/style/index.less

```diff
@@ -165,15 +164,15 @@
     min-width: 111px;
     height: 180px;
     margin: 0;
-    padding: 0;
+    padding: @cascader-dropdown-edge-child-vertical-padding 0;
     overflow: auto;
     vertical-align: top;
     list-style: none;
     border-right: @border-width-base @border-style-base @border-color-split;
     -ms-overflow-style: -ms-autohiding-scrollbar; // https://github.com/ant-design/ant-design/issues/11857
 
     &:first-child {
-      border-radius: @border-radius-base 0 0 @border-radius-base;
+        border-radius: @border-radius-base 0 0 @border-radius-base;
```

Comment:
indent is unneccessary.

Reply:
👍

Reply:
@afc163 this should be resolved resolved